docs/error-codes.md

```diff
@@ -1569,6 +1569,23 @@ void TestMethod(MethodInfo methodInfo)
 }
 ```
 
+#### `IL2104`: Type name strings used for dynamically accessing a type must be assembly qualified.
+
+Type name strings representing types with dynamically accessed members must be assembly qualified, otherwise the linker will not be able to figure out from which assembly the type should be resolved.
```

Comment:
It'd be nice to document the look up logic order, e.g. first is current assembly and then corelib.